frontend/js/src/settings/resetlatestimportts/ResetLatestImports.tsx

```diff
@@ -37,26 +37,27 @@ export default function ResetImportTimestamp() {
   return (
     <>
       <Helmet>
-        <title>Reset import timestamp</title>
+        <title>Reset your import timestamp</title>
       </Helmet>
-      <h3 className="page-title">Reset Import Timestamp</h3>
+      <h3 className="page-title">Reset your import timestamp</h3>
       <p>Are you sure you want to reset your token? </p>
```

Comment:
This is wrong, probably copy-pasted from another page. 
```suggestion
      <p>Are you sure you want to reset your timestamp? </p>
```